src/lib.rs

```diff
@@ -1041,6 +1053,7 @@ pub mod pallet {
 				staked_balance.saturating_sub(amount),
 			)?;
 			Self::add_to_release_queue(&who, amount, T::StakeUnlockDelay::get())?;
+			Self::adjust_autocompound_percentage(&who);
```

Comment:
Shouldn't this also be used for unstake? Meaning, if a staker's total stake goes below the autocompaunding threshold after executing `unstake()` or `unstake_all()` the autocompounding should also disable.

Reply:
No, because funds are still locked, although not staked. A user should be allowed to unstake from an account and stake on another one without getting their autocompund percentage set to zero.